web/src/routes/PoisPage.tsx

```diff
@@ -51,17 +64,28 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
   const formatter = useContext(DateFormatterContext)
   const { viewportSmall } = useWindowDimensions()
   const navigate = useNavigate()
-
-  if (buildConfig().featureFlags.developerFriendly) {
-    log('To use geolocation in a development build you have to start the dev server with\n "yarn start --https"')
-  }
+  const { locationState, userCoordinates } = useUserLocation()
+  const sheetRef = useRef<BottomSheetRef>(null)
 
   const requestPois = useCallback(
     async () => createPOIsEndpoint(cmsApiBaseUrl).request({ city: cityCode, language: languageCode }),
     [cityCode, languageCode]
   )
   const { data: pois, loading, error: poisError } = useLoadFromEndpoint(requestPois)
 
+  const [currentFeature, setCurrentFeature] = useState<PoiFeature | null>(null)
+  const [featureLocations, setFeatureLocations] = useState<PoiFeature[] | null>(null)
+
+  useEffect(() => {
+    if (pois && locationState.message !== 'loading') {
+      setFeatureLocations(prepareFeatureLocations(pois, userCoordinates))
+    }
+  }, [pois, locationState.message])
```

Comment:
I think we should set the `react-hooks/exhaustive-deps` linting rule to error. You are missing `userCoordinates`

Reply:
updated linting rules and missing deps in last commit